packages/patternfly-4/react-core/src/components/DataList/DataListAction.js

```diff
@@ -5,13 +5,16 @@ import styles from '@patternfly/patternfly/components/DataList/data-list.css';
 import { EllipsisVIcon } from '@patternfly/react-icons';
 import { Button } from '../Button';
 
-const DataListAction = ({ className, id, 'aria-label': ariaLabel, 'aria-labelledby': ariaLabelledBy, ...props }) => (
-  <div className={css(styles.dataListAction, className)} {...props}>
-    <Button variant="plain" id={id} aria-labelledby={ariaLabelledBy} aria-label={ariaLabel}>
-      <EllipsisVIcon />
-    </Button>
-  </div>
-);
+class DataListAction extends React.Component {
```

Comment:
why was this converted to a class?

Reply:
I converted these all to classes so I could render everything except the rowid prop in the HTML. The original implementation was showing the rowid prop and @jgiardino flagged it.

Reply:
If you destructure the rowid prop out of the ...props it should not show up in the HTML.
```
const DataListAction = ({ className, id, 'aria-label': ariaLabel, 'aria-labelledby': ariaLabelledBy, **rowid**, ...props })
```

Reply:
Thanks, I made the change.